src/js/outreach/views/opt-in_views.js

```diff
@@ -33,12 +34,30 @@ const OptInView = View.extend({
     </div>
     <div class="opt-in__field">
       <label class="opt-in__field-label">Your date of birth</label>
-      <input
-        type="date"
-        class="input-primary opt-in__field-input js-birth-date"
-        placeholder="Enter your date of birth"
-        value="{{ birth_date }}"
-      />
+      <div class="opt-in__field-select u-margin--r-4">
+        <select name="dob-month" class="u-margin--r-8 js-dob-month ">
```

Comment:
My think on is that we use a `<select>` element for these and let the user's mobile device os deal with the rest.